ui/package-lock.json

```diff
@@ -72,6 +72,35 @@
         "vitest": "^0.34.6"
       }
     },
+    "../../node_library/kestra-topology/KestraUI": {
```

Comment:
There shouldn't be any change in package-lock in this pr as you didn't change the package.json